src/css/_main-section.scss

```diff
@@ -0,0 +1,189 @@
+//     overflow: scroll;
+//   }
+
+//   .main-column-hydration {
+//     display: flex;
+//     flex-direction: row;
+//     height: 25%
+//   }
+
+//   .main-column-activity {
+//     display: flex;
+//     flex-direction: row;
+//     height: 50%
+//   }
+
+//   .main-column-sleep {
+//     display: flex;
+//     flex-direction: row;
+//     height: 35%
+//   }
+
+//   .infoContainer-cardContainer-card-horizontal {
+//     width: 100%;
+//     background-color: white;
+//     border: 1px solid black;
+//     display: flex;
+//     flex-direction: column;
+//     align-items: center;
+//     margin: 1%;
+//     justify-content: space-between;
+//   }
+
+//   .horizontalCard-activity-container {
+//     display: flex;
+//     flex-direction: row;
+//     justify-content: space-around;
+//     width: 100%;
+//     overflow: scroll;
+//   }
+
+//   .infoContainer-cardContainer-card-vertical {
+//     width: 100%;
+//     background-color: white;
+//     border: 1px solid black;
+//     display: flex;
+//     flex-direction: row;
+//     margin: 1%;
+//     overflow: scroll;
+//   }
+
+//   .card-today-hydration {
+//     text-align: center;
+//     background-color: #5BC8AC;
+//     height: auto;
+//     font-size: 1.5em;
+//     margin: .5em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: space-between;
+//     width: 80%;
+//   }
+
+//   .card-today-sleep {
+//     text-align: center;
+//     background-color: #EC96A4;
+//     height: auto;
+//     font-size: 1.5em;
+//     margin: .5em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: space-between;
+//     width: 80%;
+//   }
+
+//   .card-today-activity {
+//     text-align: center;
+//     background-color: #DFE166;
+//     height: auto;
+//     font-size: 1.5em;
+//     margin: .3em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: space-between;
+//     width: 40%;
+//   }
+
+//   .card-history-hydration {
+//     text-align: center;
+//     background-color: #5BC8AC;
+//     font-size: 1em;
+//     margin: .5em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: flex-start;
+//     width: 50%;
+//   }
+
+//   .card-history-sleep {
+//     text-align: center;
+//     background-color: #EC96A4;
+//     font-size: 1em;
+//     margin: .5em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: flex-start;
+//     width: 50%;
+//   }
+
+//   .card-history-activity {
+//     text-align: center;
+//     background-color: #DFE166;
+//     font-size: 1em;
+//     margin: .5em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: flex-start;
+//     width: 50%;
+//     height: 93%;
+//     overflow: scroll;
+//   }
```

Comment:
Is all this commented out code not being used anymore?

Reply:
Yes, I was keeping it as refrence for the older list if we needed to revert backwards